utilities/constants.py

```diff
@@ -1001,3 +1001,6 @@ class NamespacesNames:
 }
 
 ARQ_QUOTA_HARD_SPEC = {**QUOTA_FOR_POD, **QUOTA_FOR_ONE_VMI}
+
+# High performance & Numa related constants
+NODE_HUGE_PAGES_1GI_KEY = "hugepages-1Gi"
```

Comment:
This module needs to be splitted. Adding line 1006 is a smell that things got out of control here.